packages/selinux-policy/subject.cil

```diff
@@ -75,9 +75,13 @@
   network_t clock_t bus_t runtime_t
   container_t control_t super_t))
 
+; Subjects that are treated as a privileged part of the OS.
```

Comment:
I think it would help to explain the intended (high-level) difference between privileged and trusted in the comments - it seems trusted is privileged *plus* some superpowers.

Reply:
The high-level explanation tends toward a summary of the specific rules in `rules.cil`, since that's where all these sets are consumed. I am not sure how to describe it here without duplicating comments there, with the usual risk that the comments will diverge from the implementation. I'll keep thinking about it, though.